src/main/java/org/spongepowered/api/util/command/source/ProxySource.java

```diff
@@ -28,14 +28,21 @@
 
 /**
  * Proxy sources are {@link CommandSource}s that are run by one source as a
- * different source.
+ * different object.
  */
-public interface ProxySource extends LocatedSource {
+public interface ProxySource extends CommandSource {
```

Comment:
It's for those cases where the target is not in a location. There was a conversation in an "outdated diff" about that.